fix return issue

svoits opened the pull request.

svoits committed:
fix return issue

therealtoresto commented:
Hi, thanks for your contribution, but I can`t understand where u make changes for fix this problem. 
Best practices is to make 2 PR:
 1. Fix code style
 2. Fix bug with `return`
 Make that, please

svoits committed:
update

svoits committed:
update

svoits commented:
Hi Taras. 
1. fixed code styles. They were because of formatting settings.
2. Now you can see only changed lines. Also added 'y' as an answer for the question if we want to continue playing the game.
